index.html

```diff
@@ -13,7 +13,7 @@
       <div class="wrapper">
          <h1>New Identity, Who Dis?</h1>
          <p class="instructions">You ever been deep in an ex's Insta and "accidentally" like a photo from four years ago? You know, back when you were happy? Rough. That's where we come in.<br> <span class="important">You're officially a few clicks away from a new online identity. Start below.</span> </p>
-      </div>
+      </div> <!--wrapper ends here-->
```

Comment:
YES - I never usually do this in my projects, but it is a habit I should get into, because it does make it so much more readable!